### PR TITLE
fix(admin): make config changes to pass through gog middlewares (#8014)

### PR DESCRIPTION
**Pull request by joshua-goldstein**
Cherry pick from #8014 

In #7955 the team made a change to the way lambda scripts were loaded into Dgraph.  This PR had a small conflict with that PR.  Any change related to #7955 should be made in another PR.

### CODE REVIEW

**Comment by MichelDiz** [comment]
@joshua-goldstein Please, sign the CLA.

 FYI. Yes, we need getLambdaScript 100%. It is used in the cloud. If you remove it it will break the entire cloud.
What happened to make you wonder? Is it breaking some test? or CI?

**Comment by joshua-goldstein** [comment]
> @joshua-goldstein Please, sign the CLA.
> 
> FYI. Yes, we need getLambdaScript 100%. It is used in the cloud. If you remove it it will break the entire cloud. What happened to make you wonder? Is it breaking some test? or CI?

@MichelDiz I have already signed the CLA, it is giving this message because this is a cherry pick from other commits that have other contributors.

Also to your main point: this is currently not in the main branch.  [See here.](https://github.com/dgraph-io/dgraph/blob/b975b98f7671409d7b28ccd948c0bb35dab771b5/graphql/admin/admin.go#L502-L514)

**Comment by MichelDiz** [comment]
I see, yep they have changed a lot of places https://github.com/dgraph-io/dgraph/pull/7955
Also need to continue the change in the lambda repo.

In 21.03 (and previous versions) we stored the lambda scripts directly via flag in the binary. This change to GraphQL makes easy to deal with the JSscript(it is stored in bas64). This seemed obvious to me. It's a change from 16 months ago. I assumed it was already in effect.

As you can see, it is present in v21.09-slash(a cloud version) https://github.com/dgraph-io/dgraph/blob/release/v21.09-slash/graphql/admin/admin.go

File: graphql/admin/admin.go [hunk 1]
```diff
@@ -502,9 +502,10 @@ var (
 	adminQueryMWConfig = map[string]resolve.QueryMiddlewares{
 		"health":       minimalAdminQryMWs, // dgraph checks Guardian auth for health
 		"state":        minimalAdminQryMWs, // dgraph checks Guardian auth for state
-		"config":       stdAdminQryMWs,
+		"config":       gogQryMWs,
 		"listBackups":  gogQryMWs,
 		"getGQLSchema": stdAdminQryMWs,
+		//	"getLambdaScript": stdAdminQryMWs,
```
Comment (joshua-goldstein) [inline graphql/admin/admin.go:508]: this is unrelated to the PR being cherry-picked (see discussion in PR).  Adding comment to preserve this discussion for context

Comment (joshua-goldstein) [reply graphql/admin/admin.go]: > I see, yep they have changed a lot of places #7955 Also need to continue the change in the lambda repo.
> 
> In 21.03 (and previous versions) we stored the lambda scripts directly via flag in the binary. This change to GraphQL makes easy to deal with the JSscript(it is stored in bas64). This seemed obvious to me. It's a change from 16 months ago. I assumed it was already in effect.
> 
> As you can see, it is present in v21.09-slash(a cloud version) https://github.com/dgraph-io/dgraph/blob/release/v21.09-slash/graphql/admin/admin.go

That PR link is useful, thanks for the context @MichelDiz .  I am removing this line since this change is unrelated to the lambda changes in the other PR.

**Review by matthewmcneely — APPROVED** [review]

**Review by MichelDiz — APPROVED** [review]

**Review by meghalims — APPROVED** [review]
LGTM. Please merge this Josh
